backend/api/Pims.Api.csproj

```diff
@@ -2,8 +2,8 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <UserSecretsId>0ef6255f-9ea0-49ec-8c65-c172304b4926</UserSecretsId>
-    <Version>1.0.0-15.20</Version>
-    <AssemblyVersion>1.0.0.15</AssemblyVersion>
+    <Version>1.1.1-15.20</Version>
+    <AssemblyVersion>1.1.1.15</AssemblyVersion>
```

Comment:
Were these missed by the lower level updates (dev->test->master)?

Reply:
@FuriousLlama there are ongoing conversations on how to handle "hotfix" releases re: version numbering. Can discuss more on our dev meeting today

Reply:
Correct, we should have incremented IS14 to 1.1 when it was determined that IS14 was the 1.1 RC.